.github/workflows/aggkit-e2e-single-chain.yml

```
@@ -123,7 +123,7 @@ jobs:
               bats ./tests/aggkit/bridge-e2e.bats
```

```suggestion
              bats ./tests/aggkit/bridge-e2e.bats ./tests/aggkit/bridge-sovereign-chain-e2e.bats
```
Shouldn’t this file be located here for op stack?

---

Yes, it was a typo made in a rush. Thanks for pointing that out! :saluting_face: